spec/game_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../lib/game'
+
+describe Game do
+  subject(:game) { described_class.new }
+  it 'plays 10 gutter balls' do
+    20.times { game.roll(0) }
+    expect(game.score).to eq 0
+  end
+
+  it 'keeps track of a score with no bonuses' do
```

Comment:
love that you use the concept of "bonus" points - it makes it easy to track what is a "normal" (or "open") frame from the more peculiar frames.